src/style/App.css

```diff
@@ -12,3 +97,25 @@ li {
 .priority-urgent {
   color: red;
 }
+
```

Comment:
Could we change the colors for `important` and `urgent`? They don't look I think. Maybe we can choose a "better" red and a "better" yellow?